WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3482,9 +3482,9 @@ public void onMediaUploadCancelClicked(String localMediaId) {
     @Override
     public void onMediaDeleted(String localMediaId) {
         if (!TextUtils.isEmpty(localMediaId)) {
-            if (mShowAztecEditor) {
-                mAztecBackspaceDeletedMediaItemIds.add(localMediaId);
-                UploadService.setDeletedMediaItemIds(mAztecBackspaceDeletedMediaItemIds);
+            if (mShowAztecEditor && !mShowGutenbergEditor) {
+                mAztecBackspaceDeletedOrGbBlockDeletedMediaItemIds.add(localMediaId);
```

Comment:
Can we put those 2 lines of code in some method so we can call it from both blocks of code to order to increase reusability?

Reply:
👍 addressed in 133db73583